app/uk/gov/hmrc/estates/controllers/RegisterEstateController.scala

```diff
@@ -19,108 +19,44 @@ package uk.gov.hmrc.estates.controllers
-                internalId = request.identifier,
-                response = response
-              )
-
-              Logger.info("[RegisterEstateController] Estate registration completed successfully.")
-              rosmPatternService.enrolAndLogResult(response.trn, request.affinityGroup) map {
-                _ =>
-                  Ok(Json.toJson(response))
-              }
-          } recover {
-            case AlreadyRegisteredException =>
-
-              auditService.audit(
-                event = Auditing.ESTATE_REGISTRATION_SUBMITTED,
-                registration = estatesRegistrationRequest,
-                internalId = request.identifier,
-                response = RegistrationFailureResponse(403, "ALREADY_REGISTERED", "Estate is already registered.")
-              )
-
-              Logger.info("[RegisterEstateController][registration] Returning already registered response.")
-              Conflict(Json.toJson(alreadyRegisteredEstateResponse))
-            case NoMatchException =>
-
-              auditService.audit(
-                event = Auditing.ESTATE_REGISTRATION_SUBMITTED,
-                registration = estatesRegistrationRequest,
-                internalId = request.identifier,
-                response = RegistrationFailureResponse(403, "NO_MATCH", "There is no match in HMRC records.")
-              )
-
-              Logger.info("[RegisterEstateController][registration] Returning no match response.")
-              Forbidden(Json.toJson(noMatchRegistrationResponse))
-            case x: ServiceNotAvailableException =>
-
-              auditService.audit(
-                event = Auditing.ESTATE_REGISTRATION_SUBMITTED,
-                registration = estatesRegistrationRequest,
-                internalId = request.identifier,
-                response = RegistrationFailureResponse(503, "SERVICE_UNAVAILABLE", "Dependent systems are currently not responding.")
-              )
-
-              Logger.error(s"[RegisterEstateController][registration] Service unavailable response from DES")
-              InternalServerError(Json.toJson(internalServerErrorResponse))
-            case x: BadRequestException =>
-
-              auditService.audit(
-                event = Auditing.ESTATE_REGISTRATION_SUBMITTED,
-                registration = estatesRegistrationRequest,
-                internalId = request.identifier,
-                response = RegistrationFailureResponse(400, "INVALID_PAYLOAD", "Submission has not passed validation. Invalid payload..")
-              )
+  def register(): Action[JsValue] = identifierAction.async(parse.json) {
+    implicit request => {
+      request.body.validate[RegistrationDeclaration].fold(
+        errors => {
+          Logger.error(s"[RegisterEstateController][declare] unable to parse json as RegistrationDeclaration, $errors")
+          Future.successful(BadRequest)
+        },
+        declaration => {
+          registrationService
+            .submit(declaration)
```

Comment:
Do we need any auditing or logging? (Not sure what the standard level of coverage is.)

Reply:
@jaynabonne we absolutely do, I had a conversation with Alison last week and if we can implement the same auditing that is in place for Trusts. I could work on that this morning.

Reply:
I have now pushed a commit which adds auditing @jaynabonne